operations/service_deploy_smoke.go

```diff
@@ -166,10 +166,9 @@ func smokeStartEvergreen() cli.Command {
 				if err != nil {
 					return errors.Wrap(err, "error setting up monitor client directory")
 				}
-				defer func() {
-					grip.Error(clientFile.Close())
-					grip.Error(os.Remove(clientFile.Name()))
```

Comment:
Do we want to remove after we smokeRunBinary?

Reply:
We can't remove it unless the agent monitor was responsible for removing it after it has finished running (which I would prefer not to do).

Since it's a temp file that doesn't take up a huge amount of space, it's not a big deal. The agent monitor will make this temporary file anyways once it's running and won't remove it because it's the agent's binary. We can't remove it here it will cause a race between the agent monitor attempting to curl/run the evergreen binary called `clientFile` and this removing the agent file.